docs/developers-clear-containers-install.md

```diff
@@ -0,0 +1,131 @@
+# Developers Clear Containers 3.0 Install
+
+This is not a end-user guide, this is for people that are eager to try Clear Containers 3.0 and want to build it from source and are familiar doing so. Basic programming knowlege  and be confortable to work with the Linux command line is expected.
+
+
+## Requirements
+
+  * [go 1.8.3](https://golang.org/)
+  * [glibc-static](https://www.gnu.org/software/libc/libc.html) Required to build **Virtcontainers pause**
+
+## Clear Containers 3.0 components
+
+  * [Runtime](https://github.com/clearcontainers/runtime)
+  * [Proxy](https://github.com/clearcontainers/proxy)
+  * [Shim](https://github.com/clearcontainers/shim)
+  * [Virtcontainers](https://github.com/containers/virtcontainers) (**pause** binary only)
+
+**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers) Both projects ship ``cc-proxy`` and they are not compatible with each other.
+
+There are two issues to enable ``--version`` option in ``Proxy`` and ``Shim`` so the end user can know for sure which version of these components he is running.
+
+  * [https://github.com/clearcontainers/proxy/issues/69](https://github.com/clearcontainers/proxy/issues/69)
+  * [https://github.com/clearcontainers/shim/issues/32](https://github.com/clearcontainers/shim/issues/32)
+
+## Setup the environment
+
+### Define GOPATH
+
+```bash
+$ export GOPATH=$HOME/go
+```
+
+### Create GOPATH Directory
+
+```bash
+$ mkdir -p $GOPATH
+```
+
+### Get the code
+
+```bash
+$ go get github.com/clearcontainers/runtime
+$ go get github.com/clearcontainers/proxy
+$ git clone https://github.com/clearcontainers/shim $GOPATH/src/github.com/clearcontainers
+```
+
+## Build and Install Components
+
+### Proxy
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/proxy
+$ make
+$ sudo make install
+```
+
+### Shim
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/shim
+$ ./autogen.sh
+$ make
+$ sudo make install
+```
+
+### Runtime
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/runtime
+$ make
+$ sudo make install
+```
+
+### Virtcontainers (pause binary only)
+
+This step will only install the **pause** binary included in [https://github.com/containers/virtcontainers](https://github.com/containers/virtcontainers)
+
+The `pause` binary is required to allow an "empty" pod to be created. Such a pod does not contain any containers: it simply provides the environment to allow them to be created.
```

Comment:
Please change to:
The `pause` binary is required to allow the creation of an "empty" pod. The pod does not contain any containers; it simply provides the environment to allow their creation.